lib/uknetdev/include/uk/netbuf.h

```diff
@@ -41,13 +41,20 @@
 #include <uk/refcount.h>
 #include <uk/alloc.h>
 #include <uk/essentials.h>
+#if CONFIG_LIBUKSGLIST
+#include <uk/sglist.h>
+#endif /* CONFIG_LIBUKSGLIST */
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 struct uk_netbuf;
 
+#if CONFIG_LIBUKSGLIST
+struct uk_sglist;
```

Comment:
Why is this declaration needed? You include already `<uk/sglist.h>`

Reply:
You're right. I still had some build errors even with this... I will revisit now and see the preprocessed code.

Reply:
Ok it was some circular header inclusion. Solved.